app/static/lib/conp-react/src/PipelineElement/index.js

```diff
@@ -1,95 +1,163 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 
 import ViewsIcon from "../social/ViewsIcon"
 import DownloadsIcon from "../social/DownloadsIcon"
 import ArkIdElement from "../ArkIdElement"
+import CbrainModalPipeline from "../CbrainModalPipeline";
 
-const PipelineElement = props => {
+const PipelineElement = (props) => {
   const { authorized, ...element } = props;
+
+  const [modalOpen, setModalOpen] = useState(false);
```

Comment:
```suggestion
  const [cbrainModalOpen, setCbrainModalOpen] = useState(false);
```

Reply:
just because there will be other modal windows on the page so better to name it with CBRAIN for transparency